contentpool_client.module

```diff
@@ -94,3 +95,22 @@ function contentpool_client_theme($existing, $type, $theme, $path) {
     ],
   ];
 }
+
+/**
+ * Implements hook_page_attachments_alter().
+ */
+function contentpool_client_page_attachments_alter(array &$attachments) {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node instanceof NodeInterface) {
+    if ($node->hasField('field_canonical_url') &&
+      !$node->get('field_canonical_url')->isEmpty() &&
+      isset($attachments['#attached']['html_head'])) {
```

Comment:
Why must $attachments['#attached']['html_head'] be set? It should make sure the canonical is there, regardless of whether it's set already.

From requirements:
> When a node is shown on the satellite, determine whether the canonical points to a different site. If so, output it as canonical url.


The part
>determine whether the canonical points to a different site. If so
is missing.